server/routes/todoListRouter.js

```diff
@@ -12,12 +12,17 @@ const todoListRouter = require('express').Router();
 
 // Feel free to make any changes, additions, subtractions to the above list of routes we should be writing for the database queries. These are all that I could think of at the moment.
 todoListRouter.route('/:userEmail').get((req, res) => {
```

Comment:
Actually you don't need ':userEmail', this was only to specify the path. Will only need '/' or ''

Reply:
Same with below